R/file.R

```diff
@@ -42,7 +42,7 @@
 #' file_delete("mtcars.csv")
 #' \dontshow{setwd(.old_wd)}
 #' @export
-file_info <- function(path, fail = TRUE, follow = FALSE) {
+file_info <- function(path, fail = FALSE, follow = FALSE) {
```

Comment:
Keeping this as `fail = TRUE` breaks `{devtools}`, and for the sake of doing local development in RStudio, I changed this default, but this is something that needs to be discussed further.